packages/appium/gulpfile.js

```diff
@@ -47,8 +46,7 @@ boilerplate({
   test: {
     files: ['${testDir}/**/*-specs.js']
   },
-  testTimeout: 160000,
-  preCommitTasks: ['eslint', 'once'],
```

Comment:
so we got rid of all precommit tasks or we still keep the eslint one?

Reply:
the only precommit task is now eslint, and it's defined in `package.json` only